auth/auth-core/src/main/java/com/bazaarvoice/emodb/auth/identity/AuthIdentity.java

```diff
@@ -35,12 +36,18 @@
      *         If an ID is compromised an administrator may want to replace it with a new ID without
      *         changing all internal references to that ID.
      *     </li>
+     *     <li>
+     *         User management can be performed without the identity owner divulging her authentication ID,
+     *         such as over a chat client or in an email.
+     *     </li>
      * </ol>
+     *
+     * In the original design this ID was only used internally, but over time it has become an accepted public-facing
+     * identifier for identities.  This is why for historical reasons this field is called "internalId" even though
+     * it is used as both the unique internal and external identifier for this identity.
      */
     private final String _internalId;
 
```

Comment:
The reason I didn't change it is because the change cascades down.  Notably, to be consistent I'd have to change [Subject.java](https://github.com/bazaarvoice/emodb/blob/master/auth/auth-core/src/main/java/com/bazaarvoice/emodb/auth/jersey/Subject.java) and [InternalAuthorizer.java](https://github.com/bazaarvoice/emodb/blob/master/auth/auth-core/src/main/java/com/bazaarvoice/emodb/auth/InternalAuthorizer.java), as well as any classes which use them.  I agree with your statement and don't mind doing it, but it makes this PR all the larger.